gems/smithy-xml/lib/smithy-xml/codec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Xml
+    # @api private
+    class Codec
+      # @param [Hash] options
+      def initialize(options = {})
+        @options = options
```

Comment:
Similar to `json_name` setting - I'm thinking `xml_name` could be a possible setting.

Reply:
Yes, but I don't think we need it right now. For rest json and json rpc we do need that distinction.